readme.md

```diff
@@ -334,12 +334,9 @@ supported for media type `application/json`.
 
 Flow control in Conjure is a collaborative effort between servers and clients.
 
-Clients estimate the server's queue size using response delay and proactively throttle outgoing requests to minimize bottleneck queueing.  Servers may also advertise an overloaded state using 429/503 responses, which clients interpret with more drastic throttling. This results in a mechanism similar to TCP/IP flow-control and minimizes both client and server-side errors.
+Servers advertise an overloaded state using 429/503 responses, which clients interpret by throttling the number of in-flight requests they will send (currently according to an additive increase, multiplicative decrease based algorithm). Requests are retried a fixed number of times, scheduled with an exponential backoff algorithm.
```

Comment:
probably worth while noting the factors that we use, or that we proxy the defaults from netflix

Reply:
given the previous description went stale so fast, I'd prefer we keep this to minimal defaults.